bdk-swift/Package.swift.txt

```diff
@@ -0,0 +1,36 @@
+// swift-tools-version:5.5
```

Comment:
Is one of those files (the `Package.swift` and the `Package.swift.txt`) autogenerated from the other, or are they both written manually and should be kept in mind in review (in that I assume they should be perfectly in sync)?

Reply:
They should be in sync, the `Package.swift.txt` file is more of a template with variable substitution that will be done during the release workflow in the `bdk-swift` repo.